src/libsyntax/parse/mod.rs

```diff
@@ -348,20 +343,15 @@ pub fn raw_str_lit(lit: &str) -> String {
 
     // FIXME #8372: This could be a for-loop if it didn't borrow the iterator
```

Comment:
This comment can be removed now? 😊 (#8372 is closed by the introduction of `while let`)